build.gradle.kts

```diff
@@ -27,6 +27,7 @@ plugins {
     id("com.android.application") apply false
     id("com.avito.android.instrumentation-tests") apply false
     id("com.avito.android.impact")
+    id("com.avito.android.build-verdict")
```

Comment:
I don't get how to customize this block with `if(ci) id("build-verdict")

Reply:
you simply don't )